deploy/cr.yaml

```diff
@@ -723,3 +723,19 @@ spec:
 #        storageName: s3-us-west
 #        compressionType: gzip
 #        compressionLevel: 6
+#      - name: weekly-s3-us-west-incremental
+#        enabled: false
+#        schedule: "0 1 * * *"
+#        keep: 5
```

Comment:
@pooknull please check how the retention works for incremental backups. We can implement it in next release but now we need to remove "keep"  option from incremental.